app/static/js/event_list.js

```diff
@@ -5,8 +5,30 @@ $(document).ready(function(){
     $("#rsvpBtn").click(function(){
         rsvpForEvent($("#rsvpBtn").val())
     })
+    $("#toggleButton").on("click", toggleButton)
```

Comment:
What does this do? There is no `toggleButton` function to be used as an event handler

Reply:
Oversight: ...was used when a toggle button function was defined as a separate function